src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
just remove the file
we will create it when doing optionals

Reply:
ok